tools/wptrunner/wptrunner/environment.py

```diff
@@ -50,16 +49,47 @@ class TestEnvironmentError(Exception):
     pass
 
 
+def get_server_logger():
+    logger = get_default_logger(component="wptserve")
+    log_filter = handlers.LogLevelFilter(lambda x: x, "info")
+    # Downgrade errors to warnings for the server
+    log_filter = LogLevelRewriter(log_filter, ["error"], "warning")
```

Comment:
Their omission here matches the existing behaviour, and I think is desired because we rarely raise critical errors and they tend to suggest something is very wrong.

Reply:
OK, if this was existing behavior (or at least *intended* behavior given that it's current broken) then let's leave that.